--- a/application/qmodem_sipd/src/sip_consumer.c
+++ b/application/qmodem_sipd/src/sip_consumer.c
@@ -270,7 +270,6 @@ static pj_status_t lookup_credential(pj_pool_t *pool, const pj_str_t *realm,
 	credential->username = pj_str(app.username);
 	credential->data_type = PJSIP_CRED_DATA_DIGEST;
 	credential->data = pj_str(app.ha1);
-	credential->algorithm_type = PJSIP_AUTH_ALGORITHM_MD5;
 	return PJ_SUCCESS;
 }
